health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -504,7 +504,7 @@ export const UICustomizations = {
       const FacilityPopUp = Digit.ComponentRegistryService.getComponent("FacilityPopup");
       switch (key) {
         case "MICROPLAN_FACILITY_ASSIGNED_VILLAGES":
-          const assignedVillages = row?.additionalDetails?.assignedVillages;
+          const assignedVillages = row?.serviceBoundaries;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)